airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -29,11 +29,14 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
-      2.0.0:
+      4.0.0:
         message: >-
-          This version eliminates the Property History stream in favor of creating 3 different streams, Contacts, Companies, and Deals, which can now all fetch their property history.
-          It will affect only users who use Property History stream, who will need to fix schema conflicts and sync Contacts Property History stream instead of Property History.
-        upgradeDeadline: 2024-01-15
+          This update brings extended schema with data type changes for the streams Deals Property History and Companies Property History. Users will need to refresh their schema and reset their streams after upgrading.
```

Comment:
This update brings extended schema with data type changes for the streams `Deals Property History` and `Companies Property History`. Users will need to refresh their schema and reset their streams after upgrading.

Reply:
updated